doc/api/fs.md

```diff
@@ -291,6 +291,44 @@ explicitly synchronous use libuv's threadpool, which can have surprising and
 negative performance implications for some applications, see the
 [`UV_THREADPOOL_SIZE`][] documentation for more information.
 
+## class: fs.FD
```

Comment:
I’d still go by my earlier point that this should ideally not be called `FD` … `File` might already be taken but I’d still kinda prefer something that would be more intuitive to people who haven’t done much UNIX/systems programming…

Reply:
I'm open to suggestions :) I'm not good at the naming of things lol. Once had a cat named Cat.

Reply:
@nodejs/collaborators Ideas? :)

Reply:
`FileDescriptor`, `FileHandle`

Reply:
Let's go with `FileHandle`